### PR TITLE
[Admin] Add-products

### PR DESCRIPTION
**Pull request by hatcaiday**
https://edu-redmine.framgia.vn/issues/5026

### CODE REVIEW

**Comment by hatcaiday** [comment]
ready

File: app/assets/stylesheets/admin.scss [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  float: left;
+  height: 15px;
+  margin-right: 5px;
+}
+
+.new-form form button {
+  margin-top: 23px;
+}
+
+.new-form form button {
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/admin.scss]: dòng 126-172, em sửa lại dùng Nested SASS nhé

File: app/assets/stylesheets/footer.scss [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  margin-left: 15px;
+}
+
+.footer-bottom p span a {
+  color: $black;
+  font-style: italic;
+  text-decoration: underline;
+}
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/footer.scss]: end file

File: app/views/admin/users/index.html.erb [hunk 3]
```diff
@@ -22,4 +22,4 @@
       </div>
     <% end %>
   </div>
-<%= paginate @users, theme: "twitter-bootstrap-3" %>
+<%= paginate @users, theme: "twitter-bootstrap-3" %>
```
Comment (huongnguyenmta) [inline app/views/admin/users/index.html.erb]: end file

File: config/initializers/assets.rb [hunk 4]
```diff
@@ -7,6 +7,9 @@
 # Rails.application.config.assets.paths << Emoji.images_path
 # Add Yarn node_modules folder to the asset load path.
 Rails.application.config.assets.paths << Rails.root.join('node_modules')
+Rails.application.config.assets.precompile += %w( admin.css )
+Rails.application.config.assets.precompile += %w( header.css )
+Rails.application.config.assets.precompile += %w( footer.css )
```
Comment (huongnguyenmta) [inline config/initializers/assets.rb]: gộp 3 dòng 11, 12, 13

**Comment by huongnguyenmta** [comment]
commented

**Comment by hatcaiday** [comment]
ready

**Pull request merged**